src/cljdoc/render/sidebar.clj

```diff
@@ -59,40 +63,38 @@
      [:div.mb4
       (searchset-search/searchbar version-entity)]
 
-     ;; Special documents (Readme & Changelog)
-     (when (seq readme-and-changelog)
-       [:div.mb4
-        (articles/doc-tree-view version-entity readme-and-changelog (:doc-slug-path route-params))])
-
-     ;; Remaining doctree or note if missing
-     (cond
-       ;; custom doctree has been provided and so we can assume the authors are aware of
-       ;; cljdoc's articles feature -> no further notes required
-       (seq doc-tree-with-rest)
-       [:div.mb4.js--articles
-        (layout/sidebar-title "Articles" {:separator-line? (seq readme-and-changelog)})
-        [:div.mv3 (articles/doc-tree-view version-entity doc-tree-with-rest (:doc-slug-path route-params))]]
+     [:div.mb4.js--articles
+      (layout/sidebar-title [:span "Articles"
+                             (when articles-tip?
+                               [:sup.ttl
+                                [:a#js--articles-tip-toggler {:href "#"}
+                                 [:span.link.dib.ml1.v-mid.hover-blue.ttl.silver "tip"]]])])
```

Comment:
using "tip" as the link text is alright except it doesn't look clickable as-is. could we add an underline to it? and maybe make it blue like other links? I think that would make it look like some something you can interact with.

Reply:
Another common pattern is to use info/question icons like [this](https://fontawesome.com/icons/circle-info?s=solid&f=classic) or [this](https://fontawesome.com/icons/circle-question?s=regular&f=classic) instead of text, although maybe since this is a sort of warning it would make sense to use a [bang](https://fontawesome.com/icons/circle-exclamation?s=solid&f=classic)? That typically tells people there's something to interact with there. An underline + blue text would be fine, too, obviously, but I wanted to give you options. 😄

Reply:
Good points, thanks!

Cljdoc's link style does not use underlines, but the blue idea is a good thought. I just didn't want it to stand out too much. One thing I did not mention is that the tip does have a hover effect.

I did spend time searching around for icons, but in the end, thought that the word "tip" was more obvious.

Reply:
Yeah, that seems to be the sticking point with these sorts of things, making it look clickable but also not the main thing to look at. Often in cases like that I put the clickable thing to the right away from the main subject of the section, but we have that strike-through-ish line to contend with.

Reply:
Oh, it might make sense to make hover on links add an underline, too. I suppose I could make a PR for that, though.

Reply:
I think it was a design style choice. I'm ok with no underline.

Reply:
Noticing some links have a subtle hover effect, could be more consistent, I suppose.

The only links I noticed with underlines seem to be links to build errors (see shots above).

Reply:
Oh but maybe an underline on hover? That might be nice... (maybe what you meant?)

Reply:
Ahhh I see that, I hadn't realized we had an example in the same screenshot 😂

Reply:
For the hover behavior I was thinking about how github does it: https://github.com/clojure/clojure

If you hover over the repo links at the top, or any of the links in the readme, they get underlines. It feels like a subtle confirmation of "you can click this" without cluttering up the display of text generally.

Reply:
Ya, sounds good to me!

Reply:
Sometimes I feel like design is just a thousand little nudges like that which work together to give people intuition. I like the way github does it there. You can contrast it with what they do in their own docs, where information is less dense and people are generally scanning, where interlinking between docs is super important, they have underlines always on: https://docs.github.com/en/get-started/onboarding/getting-started-with-your-github-account

Reply:
But then in the side where links are more dense they use different hover effects to make it feel clickable. I wonder if they A/B tested this stuff? Sounds fun :)

Reply:
Hmmm... interesting!